### PR TITLE
Updated readme github buttons

### PR DESCRIPTION
**Pull request by swathiswaminathan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,17 @@ This demonstration will create a number of resources.
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```
Comment (emailbob) [inline README.md]: [Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)

Comment (swathiswaminathan) [reply README.md]: please test this

Comment (emailbob) [reply README.md]: Looks great

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Pull request merged**